tests/kernel/gen_isr_table/src/main.c

```diff
@@ -32,6 +32,14 @@ extern u32_t _irq_vector_table[];
 #define ISR5_OFFSET	4
 #define ISR6_OFFSET	5
 
```

Comment:
Is there any other way to fixing a problem rather than adding board-specific dependencies in generic tests? This significantly complicates maintenance of both test and boards involved.

Reply:
This test needs target-specific irq info.  Using the last several irq makes senses for most targets as they are rarely used.  EMSDP is an exception whose USB-JTAG use 108/107 irq.  In some way, it's not a good design in hardware.